<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>721</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>startGame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>pauseGame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>continueGame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>restartGame</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="startTool">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pauseTool">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QToolButton" name="continueTool">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>conti</string>
    </property>
   </widget>
   <widget class="QToolButton" name="restartTool">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>res</string>
    </property>
   </widget>
   <widget class="QToolButton" name="quitTool">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QToolButton" name="loadTool">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QToolButton" name="saveTool">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>721</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>quitGame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>loadGame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>saveGame</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionstartGame"/>
    <addaction name="actionpauseGame"/>
    <addaction name="actioncontinueGame"/>
    <addaction name="actionrestartGame"/>
    <addaction name="actionquitGame"/>
   </widget>
   <widget class="QMenu" name="menuLoad_Save">
    <property name="title">
     <string>Load/Save</string>
    </property>
    <addaction name="actionloadGame"/>
    <addaction name="actionsaveGame"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuLoad_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionstartGame">
   <property name="text">
    <string>startGame</string>
   </property>
  </action>
  <action name="actionpauseGame">
   <property name="text">
    <string>pauseGame</string>
   </property>
  </action>
  <action name="actioncontinueGame">
   <property name="text">
    <string>continueGame</string>
   </property>
  </action>
  <action name="actionrestartGame">
   <property name="text">
    <string>restartGame</string>
   </property>
  </action>
  <action name="actionquitGame">
   <property name="text">
    <string>quitGame</string>
   </property>
  </action>
  <action name="actionloadGame">
   <property name="text">
    <string>loadGame</string>
   </property>
  </action>
  <action name="actionsaveGame">
   <property name="text">
    <string>saveGame</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
